packages/console/src/command-history/CommandHistory.jsx

```diff
@@ -13,14 +13,16 @@ import {
   vsPlay,
   vsTerminal,
 } from '@deephaven/icons';
+import { StorageTableViewportUpdater } from '@deephaven/storage';
```

Comment:
This seems unused...  Looks like the eslint config is missing for the console package.

Reply:
Nice catch!